dev/vision-n-scope/7-contributions.md

```diff
@@ -12,7 +12,40 @@ placeholder
 
 ## Carter MacNab
 
-placeholder
+### Introduction
+
+ By the end of Capstone, I'd like to have worked on and completed the player client and scoreboard software necessary for the functionality of our project. I also want to play a role in observing and implementing rigid security rules to ensure our project stays free of potential vulnerability concerns. Developing the player client and focusing on security concerns will allow me to play a role in advising and learning from my fellow teammates during the course of development. 
+### System Architecture
+
+#### Figure 1:
+
+At a high level, Juno will be capable of hosting and deploying attack and defense exercises. The player will be able to connect and be administered a custom environment from which to practice these skills. This environment is made possible through communication between the game, environment, and player info server. 
+
+
+#### Figure 2:
+
+My focus will be on completing would be the player client and scoreboard. While separate, these two portions of Juno are some of the most important as they are the largest medium from which the players interact with the rest of the system. The player client ensures the player is properly connected and ensures that the images and scoreboard information is sent accordingly. 
+
+### User Stories
+“As a user, I need more engaging ways to practice and learn about key cybersecurity concepts so that I can strengthen my skills in attack and defense.” 
+
+“As a developer, I need easy ways to edit the environments served to the players so that I can more focus on tailoring a custom environment for learning rather than deployment.”
+
+“As an educator, I need a quick way to deploy environments that demonstrate key security concepts so that I can more efficiently teach my curriculum.”
+
+### Iteration Plan and Estimates
+
+Iteration 0: (1 month)
+ Write SSH shell for sending heartbeats and receiving Wireguard configs. Write tests as component is developed.
+
+Iteration 1: (1 month)
+Write program that sends heartbeats and receives Wireguard configs. Upon receiving a config, puts config in appropriate file and instantiates it. Write tests.
+
+Iteration 2: (3 weeks)
+Write the scoreboard that polls services and accepts flags. Write tests.
+
+Prep (2 weeks):
```

Comment:
since its not related to the actual implementations, i would probably leave this out